packages/presentation/presentation_publishing/interfaces/has_force_refresh.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export interface HasForceRefresh {
+  forceRefresh: () => void;
+}
+
+export const apiHasForceRefresh = (api: unknown | null): api is HasForceRefresh => {
+  return Boolean(
+    api &&
+      (api as HasForceRefresh).forceRefresh &&
```

Comment:
Is `(api as HasForceRefresh).forceRefresh` check needed? `typeof (api as HasForceRefresh).forceRefresh === 'function'` will return false when `forceRefresh` is undefined